### ISSUE TITLE
Rev2 Piunora Pro to make the most of CM5?

### ISSUE THREAD

**Issue by phil-gg** [Diodes-Delight/Piunora-Hardware#2]
Hi timonsku,

Love the Piunora Pro - I have one with a CM4 now.

I was just wondering, is there any chance you will have time to update Piunora Pro to a revision 2 board, that makes the most of the CM5?

I know that the CM5 is backwards compatible with boards like Piunora Pro (rev1), but there are small updates that could be made to make it even better for CM5, like adding extra traces to take USB from 2 to 3.

What else could be upgraded to make better use of the CM5?

Thanks again,

Phil

**Comment by timonsku** [comment]
Yes it might get updated but no near term plans for a new release as there is still a good amount rev. 1 stock.
Once that depletes a new production run would see a revision update for non breaking changes like breaking out USB3.

**Comment by phil-gg** [comment]
Thanks so much for the swift, clear response timonsku.

Some follow-up requests:

(1) Would you mind labelling this issue as "enhancement", but otherwise leave this ticket open (so others with the same query can still see your answer)?

(2) When stock gets suitably low for revision 1, would you mind coming back here, and posting again when you are ready to say "Stock is low, and I am now starting to work on CM5 enhancements for the Piunora carrier board"?  (I'll keep subscribed to this issue, and I'm sure others will too!)

(3) Are you willing to give one current stock status here now, along the lines of "As of Dec-2024, there are X lite and Y pro revision 1 boards still to sell"?

Many thanks,

Phil

**Comment by timonsku** [comment]
I have about 800 Pro units in factory stock but distributors also still have a fair amount of stock which I don't have a clear overview of as that tends to get resold to others too. Lite has not been produced again and would likely be deprecated with a new revision as the demand was not quite there even at the height of things when CM4 was newly released.
I did not have the capacity for much marketing since the launch so that likely also contributes to slow sales atm.

Can definitely ping here once a rev2 becomes more tangible and would likely send out an update to the CrowdSupply campaign whenever there is a rev2 in the works which would likely see a second CS campaign.
